js/demo/lazy-loading.js

```diff
@@ -45,7 +45,7 @@ var EffecktLazyLoading = {
   _scrollPage: function() {
 
     var self = this;
-    
+
```

Comment:
These changes are perfectly fine but they don't deserve here. This PR is meant to fix a specific ticket and shouldn't touch anything more. Same thing as https://github.com/h5bp/Effeckt.css/pull/313#discussion_r31379462. Let's keep the history/diffs clean and on-topic.